data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -187,15 +186,13 @@ We can write anonymous functions and immediately apply them to a value:
 
 ### Functions with Multiple Parameters and Partial Application
 
-A function may take several arguments, separated by spaces. This is the case both in function declaration and in function calls.
-
+A function may have several parameters, separated by spaces.
 ```ocaml
 # let cat a b = a ^ " " ^ b;;
 val cat : string -> string -> string = <fun>
 ```
 
-The function `cat` takes two `string` arguments, `a` and `b`, and returns a value of type `string`.
-
+The function `cat` has two `string` parameters, `a` and `b`, and returns a value of type `string`.
```

Comment:
Perhaps we should change this example away from "a" and "b" as we've been doing to avoid confusion with `'a` and `b`.

Reply:
Christina I think you might be looking at an older commit. At one point I kind of screwed up my merging and then didn't realize my editor had auto-formatted some things when testing things out ... In the latest commit the only affected file should be tutorial.ml

Reply:
Oh! That makes sense! I thought we had caught them all. :) 
Can you disregard that or should I delete it?

Reply:
Hmmm... will it work if I just click "resolve conversation"? 

(I haven't worked on a big team for a while and forget how this is supposed to go 😅)

Reply:
Absolutely. Just resolved it, so it should just stay the same. 🙂 
No worries about forgetting...I forget from day to day!